zio-http/shared/src/main/scala/zio/http/endpoint/Endpoint.scala

```diff
@@ -575,7 +575,7 @@ final case class Endpoint[PathInput, Input, Err, Output, Auth <: AuthType](
       output = ((HttpCodec.content[Output2] ++ StatusCodec.status(status)) ?? doc) | self.output,
       error,
       codecError,
-      documentation,
+      self.documentation,
```

Comment:
the self is now not needed anymore

Reply:
Right, because the method parameter is still called `doc`.
Good catch, fixed